File: KarateKlub/KarateKlub.Api/Models/StecenaZvanjaPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
   
    public class StecenaZvanjaPregledVM
    {
        public class Row
        {
            public string ime;
            public string prezime;
            public string imeRoditelja;
            public string nazivZvanja;
            public string mjestoSticanja;
            public DateTime datumSticanja;
            public string organizator;


        }
        public List<Row> rows { get; set; }
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_trener/model/StecenaZvanjaPregledVM.java
package com.example.x.karateklub.area_trener.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class StecenaZvanjaPregledVM implements Serializable {
    public  static class Row implements  Serializable{
        public String ime;
        public String prezime;
        public String imeRoditelja;
        public String nazivZvanja;
        public String mjestoSticanja;
        public Date datumSticanja;
        public String organizator;
    }
    public List<Row> rows;
}
